src/Runtime/Distributions/Bernoulli.cs

```diff
@@ -53,7 +53,7 @@ public double GetProbTrue()
         /// </summary>
         public void SetProbTrue(double probTrue)
         {
-            if (probTrue < 0 || probTrue > 1) throw new ArgumentOutOfRangeException("probTrue", String.Format("probTrue = {0} is not in [0,1]", probTrue));
+            if (probTrue < 0 || probTrue > 1) throw new ArgumentOutOfRangeException(nameof(probTrue), String.Format("probTrue = {0} is not in [0,1]", probTrue));
```

Comment:
Funny how you changed it in one place but not the other place.

Reply:
My eyes got a little blurry, got that one and a couple more now.